cla_backend/apps/cla_butler/tests/test_diversity_key_change.py

```diff
@@ -0,0 +1,76 @@
+import os
+import mock
+from django.db import transaction
+from django.test import TestCase
+from django.db.utils import InternalError
+from core.tests.mommy_utils import make_recipe
+from legalaid.utils import diversity
+from legalaid.models import Case
+
+
+class DiversityReencryptTestCase(TestCase):
+    def get__key(self, key_name):
+        file_path = os.path.join(os.path.dirname(diversity.__file__), "keys", key_name)
+        with open(file_path) as f:
+            return f.read()
+
+    def test_diversity_key_change(self):
```

Comment:
Could split this into two tests, test diversity key change with incorrect passphrase and with correct passphrase